submissions/admin.py

```diff
@@ -1,13 +1,13 @@
+""" Submissions Admin Views. """
 from __future__ import absolute_import
 
 from django.contrib import admin
-from django.core.urlresolvers import reverse
-from django.utils import html
+from django.urls import reverse
 
 from submissions.models import Score, ScoreSummary, StudentItem, Submission
 
 
-class StudentItemAdminMixin(object):
+class StudentItemAdminMixin():
```

Comment:
This is a good improvement for Python-3-only code, but changes behavior on Python 2; it changes this from a [new-style class to an old-style class](https://docs.python.org/2/reference/datamodel.html#newstyle).